pinot-broker/src/main/java/org/apache/pinot/broker/api/resources/PinotClientRequest.java

```diff
@@ -236,6 +238,48 @@ public void processSqlWithMultiStageQueryEnginePost(String query, @Suspended Asy
     }
   }
 
+  @GET
+  @ManagedAsync
+  @Produces(MediaType.APPLICATION_JSON)
+  @Path("timeseries/api/v1/query_range")
+  @ApiOperation(value = "Prometheus Compatible API for Pinot's Time Series Engine")
+  @ManualAuthorization
+  public void processTimeSeriesQueryEngine(@Suspended AsyncResponse asyncResponse,
```

Comment:
I noticed these API's are not part of Swagger endpoint. Do we need to make additional changes to include in swagger console in pinot?

Reply:
Are you checking the pinot-controller swagger? I can take it as a follow-up.. I almost never use the pinot-broker swagger.